fdbcli/fdbcli.actor.cpp

```diff
@@ -1627,6 +1627,13 @@ ACTOR Future<int> cli(CLIOptions opt, LineNoise* plinenoise) {
 					continue;
 				}
 
+				if (tokencmp(tokens[0], "versionepoch")) {
```

Comment:
You could consider adding some tests of this to bindings/python/tests/fdbcli_tests.py